tests/Aspire.Cli.Tests/TestServices/TestConsoleInteractionService.cs

```diff
@@ -85,6 +85,11 @@ public void DisplayPlainText(string text)
     {
     }
 
+    public void WriteConsoleLog(string message, int? lineNumber = null, string? type = null, bool isErrorMessage = false)
+    {
+        Console.WriteLine($"[{(isErrorMessage ? "Error" : type ?? "Info")}] {message} (Line: {lineNumber})");
```

Comment:
Add a callback here that we can hook like in `DisplaySubtleMessage(...)`

Reply:
added